aleo/caip2.md

```diff
@@ -26,14 +26,14 @@ replaces (*optional): <CAIP-2>
 
 The namespace `aleo` refers to the Aleo Network Layer 1 blockchain.
 
-To date, Aleo consists of a single network: a testnet network (Testnet3).
+To date, Aleo consists of a single network: a testnet network (Testnet).
```

Comment:
Would it in this case reference both the `testnet` and `mainnet`?

Reply:
No, just removed this sentence.